scripts/tests/api_compare/setup.sh

```diff
@@ -19,6 +19,6 @@ docker compose rm -f
 # Ideally, we could use `--wait` and `--wait-timeout` to wait for services
 # to be up. However, `compose` does not distinct between services and 
 # init containers. See more: https://github.com/docker/compose/issues/10596
-docker compose up --build --force-recreate --detach --timestamps
+docker compose --profile offline-rpc-server up --build --force-recreate --detach --timestamps
```

Comment:
The naming is confusing. Reading this code, I'd assume that this will run _only_ the offline server.

I recommend a more explicit name, e.g., `--include-offline-rpc`

Reply:
Fixed.